spec/amp-actions-and-events.md

```diff
@@ -203,22 +204,26 @@ event.index</pre></td>
   </tr>
 </table>
 
-### amp-list
+### amp-list <a name="amp-list"></a>
 
 <table>
   <tr>
     <th width="25%">Event</th>
     <th width="35%">Description</th>
     <th width="40%">Data</th>
   </tr>
+  <tr>
+    <td><code>changeToLayoutContainer</code></td>
+    <td>Update's <code>amp-list</code>'s layout to <code>layout="CONAINTER"</code>.</td>
```

Comment:
s/container/conainter -- is this worth linking to the [Dynamic resizing](https://amp.dev/documentation/components/amp-list/#dynamic-resizing) section of amp-list or nah? It's not obvious from this language why someone would want to do this.

Reply:
wouldn't hurt!